src/common/transformations/include/transformations/common_optimizations/constants_reduce.hpp

```diff
@@ -0,0 +1,22 @@
+// Copyright (C) 2024 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include "transformations_visibility.hpp"
+#include "openvino/pass/graph_rewrite.hpp"
+
+namespace ov {
+namespace pass {
+
+class TRANSFORMATIONS_API ConstantsReduce : public ov::pass::GraphRewrite {
+public:
+    OPENVINO_GRAPH_REWRITE_RTTI("ConstantsReduce");
```

Comment:
please use `OPENVINO_MODEL_PASS_RTTI` or `OPENVINO_MATCHER_PASS_RTTI` according to the comment above